<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTreeWidget" name="db_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
     </property>
     <column>
      <property name="text">
       <string>数据库列表</string>
      </property>
      <property name="foreground">
       <brush brushstyle="Dense2Pattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
    <widget class="HighLightTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>581</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>581</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>id</string>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>rank</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>qwe</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(195, 195, 195);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(134, 134, 134);</string>
   </property>
   <widget class="QMenu" name="new_db">
    <property name="title">
     <string>主界面</string>
    </property>
    <addaction name="exit_2"/>
   </widget>
   <addaction name="new_db"/>
  </widget>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="newdb">
   <property name="text">
    <string>新建数据库</string>
   </property>
  </action>
  <action name="newtable">
   <property name="text">
    <string>新建表</string>
   </property>
  </action>
  <action name="yunx">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="exit_2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HighLightTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">highlighttextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
